File: copl/com.elster.jupiter.messaging/src/main/java/com/elster/jupiter/messaging/Message.java
/*
 * Copyright (c) 2017 by Honeywell International Inc. All Rights Reserved
 */

package com.elster.jupiter.messaging;

import aQute.bnd.annotation.ProviderType;

/**
 * Abstraction for a message on a queue.
 */
@ProviderType
public interface Message {

    /**
     * @return the raw bytes representing the payload of the message.
     */
    byte[] getPayload();
}

File: comu/com.energyict.comserver.comserver.core.extjs/src/main/web/js/mdc/src/model/Message.js
/*
 * Copyright (c) 2017 by Honeywell International Inc. All Rights Reserved
 */

Ext.define('Mdc.model.Message', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'name', type: 'string'},
        {name: 'active', type: 'boolean'},
        {name: 'privileges', type: 'auto'}
    ]
});